openedx/core/djangoapps/plugin_api/views.py

```diff
@@ -134,9 +132,7 @@ def render_standalone_response(self, request, fragment, **kwargs):
             'fragment': fragment,
             'page_title': self.standalone_page_title(request, fragment, **kwargs),
         })
-        if context.get('uses_pattern_library', False):
-            template = 'fragments/standalone-page-v2.html'
```

Comment:
Template removed.

Reply:
Am I wrong to think this means we'll never make it into the else block (both before and after this change)?

```
        else:
            template = 'fragments/standalone-page-v1.html'
```

Reply:
Yes, I think that you are right. It seems that it is not being used. I am going to remove that too.